src/main/java/DenisCamacho/task_01/ContactManager.java

```diff
@@ -0,0 +1,47 @@
+package main.java.DenisCamacho.task_01;
+
+import java.util.ArrayList;
+
+public class ContactManager {
+    ArrayList<Contact> myFriends;
+    int friendsCount;
+
+    ContactManager() {
+        this.myFriends = new ArrayList<>();
+        this.friendsCount = 0;
+    }
+
+    public void addContact(Contact contact) {
+        myFriends.add(contact);
+        friendsCount++;
+    }
+
+    public Contact searchContact(String searchName) {
+        for (Contact contact : myFriends) {
+            if (contact.getName().equals(searchName)) {
+                return contact;
+            }
+        }
+        return null;
+    }
+
+    public void showContact() {
```

Comment:
@denis1729 this method seems like is not for this class, for now it help us to print all in console just calling this method, but if you want to implement a UI this method will be useless, this behavior is related to the console